manifest/x86_64/r/ruby_power_assert.filelist

```diff
@@ -0,0 +1,2 @@
+# Total size: 0
+
```

Comment:
No files?

Reply:
I'm not sure what's breaking during this update process for these gems. I'm going to rebuild the containers and see what happens.

Reply:
Filelists are fixed!